lib/perl/Genome/Test/Command/RunPteroIntegrationTests.pm

```diff
@@ -2,14 +2,18 @@ package Genome::Test::Command::RunPteroIntegrationTests;
 
 use strict;
 use warnings FATAL => 'all';
-use Genome;
+use above "Genome";
```

Comment:
It's unusual to see `use above` in a `.pm` rather than a Perl script that's using Genome.  Is there something particular about this module that requires it?